meta-resin-common/recipes-support/resin-init/resin-init-flasher.bb

```diff
@@ -68,3 +68,9 @@ do_install() {
     echo "BOOTLOADER_BLOCK_SIZE_OFFSET=${BOOTLOADER_BLOCK_SIZE_OFFSET}" >> ${D}/${sysconfdir}/resin-init-flasher.conf
     echo "BOOTLOADER_SKIP_OUTPUT_BLOCKS=${BOOTLOADER_SKIP_OUTPUT_BLOCKS}" >> ${D}/${sysconfdir}/resin-init-flasher.conf
 }
+
+do_deploy() {
+    touch ${DEPLOYDIR}/uEnv.txt
```

Comment:
I would like to have this file with configuration of booting from sd. Which would get modified at the end of the flasher script.